polylogue/cli/verify.py

```
@@ -0,0 +1,432 @@
+        return None
+    values = {chunk.strip().lower() for chunk in str(raw).split(",") if chunk.strip()}
+    return values or None
+
+
+def _safe_json(raw: Optional[str]) -> Dict[str, Any]:
+    if not raw:
+        return {}
+    try:
+        payload = json.loads(raw)
+        return payload if isinstance(payload, dict) else {}
+    except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)